python_modules/libraries/dagster-github/dagster_github/resources.py

```diff
@@ -0,0 +1,163 @@
+            headers = {}
+        self.__check_app_token()
+        headers["Authorization"] = "Bearer {}".format(self.app_token["value"])
+        headers["Accept"] = "application/vnd.github.machine-man-preview+json"
+        request = requests.post(
+            "https://api.github.com/app/installations/{}/access_tokens".format(installation_id),
+            headers=headers,
+        )
+        request.raise_for_status()
+        auth = request.json()
+        self.installation_tokens[installation_id] = {
+            "value": auth["token"],
+            "expires": to_seconds(datetime.strptime(auth["expires_at"], '%Y-%m-%dT%H:%M:%SZ')),
+        }
+
+    def __check_installation_tokens(self, installation_id):
+        if (installation_id not in self.installation_tokens) or (
+            self.installation_tokens[installation_id]["expires"] < (int(time.time()) + 60)
+        ):
+            self.__set_installation_token(installation_id)
+
+    def execute(self, query, variables, headers=None, installation_id=None):
+        if headers is None:
+            headers = {}
+        if installation_id is None:
+            installation_id = self.default_installation_id
+        self.__check_installation_tokens(installation_id)
+        headers["Authorization"] = "token {}".format(
+            self.installation_tokens[installation_id]["value"]
+        )
+        request = requests.post(
+            "https://api.github.com/graphql",
+            json={"query": query, "variables": variables},
+            headers=headers,
+        )
+        request.raise_for_status()
+        return request.json()
+
+    def create_issue(self, repo_name, repo_owner, title, body, installation_id=None):
+        if installation_id is None:
+            installation_id = self.default_installation_id
+        res = self.execute(
+            query="""
+            query get_repo_id($repo_name: String!, $repo_owner: String!) {
+                repository(name: $repo_name, owner: $repo_owner) {
+                    id
+                }
+            }
+            """,
+            variables={"repo_name": repo_name, "repo_owner": repo_owner},
+            installation_id=installation_id,
+        )
+
+        return self.execute(
+            query="""
+                mutation CreateIssue($id: ID!, $title: String!, $body: String!) {
+                createIssue(input: {
+                    repositoryId: $id,
+                    title: $title,
+                    body: $body
+                }) {
+                    clientMutationId,
+                    issue {
+                        body
+                        title
+                        url
+                    }
+                }
+                }
+            """,
+            variables={"id": res["data"]["repository"]["id"], "title": title, "body": body,},
+            installation_id=installation_id,
+        )
+
+
+@resource(
+    config={
+        "github_app_id": Field(
+            Int,
```

Comment:
Ah so new thing that has transpired in the past two days is that you no longer have to use "Dagster Types" or typing types for native python types in fields. This makes it easier to compose. I think if you pull in the recent changes to master, this code might not work. Sorry about that!

Reply:
in other words. Replace `Int`, `String`, `Float`, etc. with `int`, `str`, `float`, etc.